config/jobs/kubernetes/sig-k8s-infra/registry.k8s.io/canaries.yaml

```diff
@@ -14,6 +14,35 @@ presets:
     value: registry-sandbox.k8s.io/pause:3.8
 
 periodics:
+# TODO: copy this job onto the EKS cluster and run from AWS once the job is working
+# and the EKS CI cluster is ready
+- name: registry-sandbox-e2e-gcp
+  cluster: k8s-infra-prow-build
+  interval: 10m
```

Comment:
if not for the time to build the test binary, these run in a few seconds.

We can make the job shallow clone actually and really speed these up so the job is a very quick + cheap run, hence bias towards frequent coverage

Reply:
set clone depth to cut time even more